tests/unit/test-cmd/test.build.js

```diff
@@ -329,4 +368,42 @@ describe('build', () => {
 
   });
 
+  describe('defaultPackageCreator', () => {
+    it('should reject on Unexpected errors', () => {
+      return withTempDir(
+        (tmpDir) => {
+          const fakeEventToPromise = sinon.spy(async (stream) => {
+            await defaultEventToPromise(stream, 'close');
+            //Remove contents of tmpDir before removal of directory
```

Comment:
Same coding style issue of one of the previous comments (Add a space between `//` and the comment and ends it with a `.`).